### PR TITLE
Get Weblate translations in

### PR DESCRIPTION
**Pull request by stanislav-brabec**
## What does this PR change?
Include translations from https://l10n.opensuse.org/projects/uyuni/

## GUI diff

No difference.

Before:

After:

- [ ] **DONE**

## Documentation
- No documentation needed: **add explanation. This can't be used if there is a GUI diff**
- Documentation issue was created: [Link for SUSE Manager contributors](https://github.com/SUSE/spacewalk/issues/new?template=ISSUE_TEMPLATE_DOCUMENTATION.md&labels=documentation&projects=SUSE/spacewalk/31), [Link for community contributors](https://github.com/uyuni-project/uyuni-docs/issues/new).
- (OPTIONAL) [Documentation PR](https://github.com/uyuni-project/uyuni-docs/pulls)

- [ ] **DONE**

## Test coverage
- No tests: **add explanation**
- Unit tests were added
- Cucumber tests were added

- [ ] **DONE**

## Links

Fixes #
Tracks # **add downstream PR, if any**

- [ ] **DONE**

## Changelogs

If you don't need a changelog check, please mark this checkbox:

- [ ] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_lint_checkstyle"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "ruby_rubocop"
- [ ] Re-run test "schema_migration_test_oracle"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

File: client/rhel/yum-rhn-plugin/po/ca.po [hunk 1]
```diff
@@ -90,9 +90,9 @@ msgstr "Missatge d'error:"
 
 #: ../rhnplugin.py:276
 msgid "Please run rhn_register as root on this client"
-msgstr "Si us plau, executeu rhn_register com a root sobre aquest client"
+msgstr "Si us plau, executeu rhn_register com a arrel en aquest client."
```
Comment (juliogonzalez) [inline client/rhel/yum-rhn-plugin/po/ca.po:93]: I am not catalonian speaker myself, but I am pretty sure catalonians do not translate the `root` user into `arrel` user. https://ca.wikipedia.org/wiki/Superusuari

@srbarrios can you have a quick look at the catalonian translations of this PR? Maybe there are some other typos.

File: client/rhel/yum-rhn-plugin/po/sk.po [hunk 2]
```diff
@@ -9,86 +9,93 @@ msgstr ""
...
+msgstr "Podpora Red Hat Satellite alebo RHN Classic bude zakázaná."
 
 #. If non-root notify user Spacewalk repo not accessible
 #: ../rhnplugin.py:70
 msgid ""
 "*Note* Spacewalk repositories are not listed below. You must run this "
 "command as root to access Spacewalk repositories."
 msgstr ""
+"*Poznámka* Repozitáre Spacewalk nie sú nižšie uvedené. Tento príkaz musíte "
+"spustiť ako správca systému, aby ste získali prístup k repozitárom Spacewalk."
```
Comment (juliogonzalez) [inline client/rhel/yum-rhn-plugin/po/sk.po:38]: Isn't it `správca systému` translated as `system administrator`? Of course I am not slovak speaker, but I wonder if it's int more clear to talk about the `root user` to make it clear.

File: client/rhel/yum-rhn-plugin/po/sk.po [hunk 3]
```diff
@@ -9,86 +9,93 @@ msgstr ""
...
 
 #: ../rhnplugin.py:275
 msgid "Error Message:"
-msgstr ""
+msgstr "Chybová správa:"
 
 #: ../rhnplugin.py:276
 msgid "Please run rhn_register as root on this client"
-msgstr ""
+msgstr "Prosím, spustite rhn_register ako správca systému na tomto klientovi"
```
Comment (juliogonzalez) [inline client/rhel/yum-rhn-plugin/po/sk.po:96]: Same here.

**Comment by stanislav-brabec** [comment]
If you want to perform review, there is a chance to force review in Weblate (you can approve/disapprove per string, not per patch).

I am not author of these translations, I am just a maintainer of openSUSE Weblate online translation web. We are just talking about a way to automate the merge of translations.

**Comment by juliogonzalez** [comment]
@stanislav-brabec I know. However maybe I am wrong and my comments can be ignored, that's why I pinged them here.

Of course if there are no issues, we can merge as it is. Otherwise this will need to get fixed at weblate, as you just said.

**Comment by juliogonzalez** [comment]
Dear developer:

A branch for SUSE Manager 4.1 is now created on the internal repository, and all package SPEC files in the master branch of this repository are now updated to be '4.2.0' version.

All developers, regardless of being SUSE employee or not, need to keep the following things in mind:

1. Please rebase this PR against `master`branch and fix any conflicts.
2. If this PR includes schema upgrades, change the directory for the migration scripts to the **correct one**. At this precise moment it is `schema/spacewalk/upgrade/susemanager-schema-4.2.0-to-susemanager-schema-4.2.1` but it could be a different one if this PR is still open after Uyuni 2020.08 is released.

SUSE employees must also keep in mind:

1. If you merge the PR here, you most likely want to backport it to the new `Manager-4.1` branch.
2. You will need to **adapt** the PR when backporting for at least the changelogs and schema migrations (if there are on your PR).

**Comment by mcalmer** [comment]
I did a merge of weblate and uyuni. So this can be closed

**Pull request closed without merging**